docs_src/path_params_numeric_validations/tutorial002.py

```diff
@@ -5,7 +5,5 @@
 
 @app.get("/items/{item_id}")
 async def read_items(q: str, item_id: int = Path(title="The ID of the item to get")):
-    results = {"item_id": item_id}
-    if q:
-        results.update({"q": q})
+    results = {"item_id": item_id, "q": q}
```

Comment:
What if `q` is an empty string?

```
$ curl http://localhost:8000/items/42?q=
```

Nothing in the annotation sets a minimum length.

Reply:
So it returns something like that
```python
{
    "item_id": 123,
    "q": "",
}
```
Why not? Just tutorial example